yowsup/layers/protocol_profiles/layer.py

```diff
@@ -13,6 +13,8 @@ def __str__(self):
     def sendIq(self, entity):
         if entity.getXmlns() == "w:profile:picture":
             self.toLower(entity.toProtocolTreeNode())
+        else:
+            self.entityToLower(entity)
```

Comment:
This will cause redundant iq messages to be sent, I'm going to change it to:
```
        elif entity.getXmlns() == "status":
            self.entityToLower(entity)
````